src/osip-monitoring-view/src/main/java/edu/kit/pse/osip/monitoring/view/settings/GeneralSettings.java

```diff
@@ -2,54 +2,141 @@
 
 import edu.kit.pse.osip.core.io.files.ClientSettingsWrapper;
 import edu.kit.pse.osip.core.model.base.TankSelector;
+import edu.kit.pse.osip.core.utils.language.Translator;
+import edu.kit.pse.osip.monitoring.view.dashboard.MonitoringViewConstants;
+import java.util.EnumMap;
+import javafx.geometry.Insets;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.Slider;
+import javafx.scene.control.Spinner;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.GridPane;
 
 /**
  * Contains all controls for setting the general settings.
+ * 
+ * @author Martin Armbruster
+ * @version 1.2
  */
-public class GeneralSettings extends javafx.scene.control.ScrollPane {
+class GeneralSettings extends ScrollPane {
     /**
      * Used for setting the servers' host name.
      */
-    private javafx.scene.control.TextField serverHostname;
+    private TextField serverHostname;
+    
     /**
      * Used for setting the server ports.
      */
-    private java.util.Collection<javafx.scene.control.TextField> serverPorts;
+    private EnumMap<TankSelector, Spinner<Integer>> serverPorts;
+    
     /**
      * Slider to set the update interval.
      */
-    private javafx.scene.control.Slider timeSlider;
+    private Slider timeSlider;
+    
     /**
      * Shows the update interval as a number.
      */
-    private javafx.scene.control.TextField timeBox;
+    private Spinner<Number> timeBox;
+    
     /**
-     * Creates a new tab that holds the general settings
+     * Creates a new tab that holds the general settings.
+     * 
      * @param currentSettings The current settings used for displaying them.
      */
-    protected GeneralSettings (ClientSettingsWrapper currentSettings) {
-        throw new RuntimeException("Not implemented!");
+    protected GeneralSettings(ClientSettingsWrapper currentSettings) {
+        serverPorts = new EnumMap<>(TankSelector.class);
+        createLayout(currentSettings);
     }
+    
     /**
-     * Returns the servers' hostname.
+     * Creates the layout for this element.
+     * 
+     * @param currentSettings The current settings.
+     */
+    private void createLayout(ClientSettingsWrapper currentSettings) {
+        Label generalLabel;
+        Translator translator = Translator.getInstance();
+        
+        GridPane layout = new GridPane();
+        layout.setHgap(MonitoringViewConstants.ELEMENTS_GAP);
+        layout.setVgap(MonitoringViewConstants.ELEMENTS_GAP);
+        layout.setPadding(new Insets(MonitoringViewConstants.ELEMENTS_GAP));
+        
+        generalLabel = new Label(translator.getString("monitoring.settings.generalSettings.updateInterval"));
+        timeSlider = new Slider(MonitoringViewConstants.MIN_UPDATE_INTERVAL,
```

Comment:
> DoubleProperty 	value
The current value represented by this Slider.

Quelle: https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/Slider.html

Der Slider rechnet mit doubles, d. h. es gibt keine andere Möglichkeit.